src/main/java/Byte/Byte.java

```diff
@@ -0,0 +1,266 @@
+package Byte;
+
+import Byte.exception.ByteException;
+import Byte.task.Deadline;
+import Byte.task.Event;
+import Byte.task.Task;
+import Byte.task.ToDo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Byte {
+    private static final int MAX_TASKS = 100;
+    private static final List<Task> tasksList = new ArrayList<>();
+    private static boolean tasksChanged = false;
+    private static final String FILE_PATH = "./data/byte.txt";
+
+    private static final int TASK_TYPE_INDEX = 0;
+    private static final int DESCRIPTION_INDEX = 2;
+    private static final int DEADLINE_TIME_INDEX = 3;
+    private static final int EVENT_START_TIME_INDEX = 3;
+    private static final int EVENT_END_TIME_INDEX = 4;
+
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        runByte(scanner);
+        if (tasksChanged){
+            try{
+                saveTasksToFile();
+            }catch (IOException e){
```

Comment:
```suggestion
            } catch (IOException e) {
```
Perhaps considering adding spaces in accordance to [java coding standard](https://se-education.org/guides/conventions/java/basic.html).

Reply:
Noted